xml_inscriptions/transcribed/NY.NY.CU.Butl.L.48.xml

```diff
@@ -0,0 +1,168 @@
+                  </origin>
+                  <provenance type="observed" subtype="reported">
+                     <p>Currently housed in the Rare Books and Manuscript Library in the <placeName>Columbia University Butler Library</placeName>
+                     </p>
+                  </provenance>
+               </history>
+            </msDesc>
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#AJA_Olcott1"/>
+                  <biblScope unit="item">158, no. 10</biblScope>
+               </bibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#CIL_VI"/>
+                  <biblScope unit="item">38394</biblScope>
+               </bibl>
+               <bibl>
+                  <ref type="external"
+                        target="http://www.edr-edr.it/edr_programmi/res_complex_comune.php?do=book&amp;id_nr=EDR114861">EDR114861</ref>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+                      href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2024-10-02" who="Norah Mezey-McMahon">Updated metadata, added transcription and translation.</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--facsimile element goes here-->
+   <text>
+      <body>
+         <div type="edition">
+         <ab>
+            <lb n="1"/>
+               <expan><abbr>D</abbr><ex>is</ex></expan>
+               <expan><abbr>M</abbr><ex>anibus</ex></expan>
+               <expan><abbr>s</abbr><ex>acrum.</ex></expan>
+            <lb n="2"/>
+               <persName>
+                  <name type="praenomen" key="Marcus">
+                  <expan><abbr>M</abbr><ex>arcus</ex></expan>
+                  </name>
+               </persName>
+               <persName>
+                  <name type="gentilicium" key="Fundanius">Fundanius</name>
+               </persName>
+               <persName>
+                  <name type="cognomen" key="Secundus">Se</name>
+               </persName>
+            <lb n="3" break="no"/>
+               cundus <expan><abbr>vi</abbr><ex>xit</ex></expan>
+               <expan><abbr>an</abbr><ex>nis</ex></expan><num value="40">XXXX;</num>
+            <lb n="4"/>
+               fecit
+               <persName>
+                  <name type="praenomen" key="Istamenia">Istamenia</name>
+               </persName>
+               <persName>
```

Comment:
This is the same person, right? No need to close the persName between the two parts of the name, the entirety should be surrounded by the element.